index.rst

```diff
@@ -6,65 +6,110 @@
 
 Overview
-================
+========
 The Auxilliary Telescope (AuxTel) Calibration Illumination System will be used to illuminate the white calibration screen in the AuxTel with a tunable light source, ranging in wavelength from 350-1200 nm. The illumination system will sit on the dome floor directly below the calibration screen. 
 
-The system was developed and built in Tucson. This document is meant to give an overview of the system and document the assembly and integration process of the system when shipped to Chile. This document also includes an overview of how the system should be operated with sample code, and links to all documentation related to this system so it can be more readily accessed. 
-
 The work to prepare this subsystem for shipment was tracked in the JIRA Epic `SITCOM-259c <https://jira.lsstcorp.org/browse/SITCOM-259>`__.
 
 Some documentation on this system can be accessed on `Docushare Collection-11943 <https://docushare.lsst.org/docushare/dsweb/View/Collection-11943>`__.
 
 There are additional details about control of the LabJack in `Tech Note 36 <https://tstn-036.lsst.io>`__.
 An overview of the data is in a Jupyter Notebook in the github for this tech note. 
 
-
-Hardware
-==============
+System Description
+==================
 There are five main functional elements of the AuxTel Illumination System:
 
-1. **The White Light Source** supplies the light, and requires a **chiller** to remove heat from the beam.
+1. **The White Light Source (WLS)** supplies the light, and requires a **chiller** to remove heat from the beam.
 2. **The Monochromator** changes the wavelength of the output light.
 3. **Optics** deliver the light to the calibration screen. There is a shutter that can be opened when using the illumination system.
 4. The **Electrometer** and **Fiber Spectrograph** are used to monitor the output beam. 
 5. The **Electronics Cabinet** provides power and control to the other hardware. 
 
 These elements are integrated on a large, heavy table. The White light source, monochromator, and optics are all housed in a light tight box. 
 
+.. figure:: /_static/full_lab.png
+ :name: full_lab
+ :target: ../_images/full_lab.png
+ :alt: full_lab
+
+ AuxTel Calibration Illumination System setup in the Tucson Lab.
+
+.. figure:: /_static/left_door.png
+ :name: left_door
+ :target: ../_images/left_door.png
+ :alt: left_door
+ :scale: 50 %
+
+ Left side of Illumination as seen in the AT dome.
+
+.. figure:: /_static/right_door.png
+ :name: right_door
+ :target: ../_images/right_door.png
+ :alt: right_door
+ :scale: 50 %
+
+ Right side of Illumination as seen in the AT dome.
+
 White Light Source
 ------------------
 The white light source is a KiloArc Lamp built by Optical Building Blocks. It can output white light with a power of 800 - 1200 W. This light is directed into the monochromator.
```

Comment:
When writing docs, it's a really good idea to use semantic line breaks.
https://sembr.org/
I'm not saying you have to go through and change everything, but use them next time as it makes life easier when doing edits.

Reply:
Great, will do.

Reply:
I just went ahead and changed the whole document this time for practice.